api/src/pcapi/core/offers/api.py

```diff
@@ -340,7 +349,7 @@ def _update_collective_offer(
     return updated_fields
 
 
-def batch_update_offers(query: BaseQuery, update_fields: dict) -> None:
+def batch_update_offers(query: BaseQuery, update_fields: dict, send_email_notif: bool = False) -> None:
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Same here :slightly_smiling_face: 
```suggestion
def batch_update_offers(query: BaseQuery, update_fields: dict, send_email_notification: bool = False) -> None:
```